### DISCUSSION TITLE
How to draw RGB images

### DISCUSSION THREAD

**Discussion by Fanchengyan** [raphaelquast/EOmaps#248] [Q&A] [answered]
Can `EOmaps` draw RGB images? I couldn't find related examples/API on the official website.

**Comment by raphaelquast** [comment, marked as answer]
Hey, yes of course it can! 
You can find some examples in the data-visualization section of the docs:

https://eomaps.readthedocs.io/en/latest/api_data_visualization.html#rgb-rgba-composites

**Reply by Fanchengyan** [reply]
Thank you very much. The painting has been successful.  I have a small suggestion, though: 

The entire tutorial lacks examples corresponding to `plt.imshow`. Therefore, I recommend that the tutorial include the use of `m.set_shape.raster()` to display a normal RGB image. Otherwise, the RGB dots might easily lead users to believe that only dots can be drawn.

![截图_20240715004303](https://github.com/user-attachments/assets/0dc562ef-0e70-4eca-b8a3-8e43470a99e7)

**Reply by raphaelquast** [reply]
Yes, i fully agree that the docs could be improved here! It might even be a good idea to make RGB/RGBA composite creation a subsection in the "Data Visualization" chapter...

 I'd be happy to accept a pull-request that adds additional information to the docs!
